independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/ResteasyReactiveConfig.java

```diff
@@ -9,7 +9,15 @@
     private long inputBufferSize;
 
     /**
-     * By default we assume a default produced media type of "text/plain"
+     * The size of the output stream response buffer. If a response is larger than this and no content-length
+     * is provided then the request will be chunked.
+     *
+     * Larger values may give slight performance increases for large responses, at the expense of more memory usage.
+     */
+    private int outputBufferSize = 8192;
```

Comment:
I think we should be consistent and not set a default value here.

Reply:
It causes problems with the standalone launcher (the one that is used when Quarkus is not in play).

Reply:
But why given you set the value in the constructor (see my other comment above)? And all the other values are not set so that should be a problem too?

Discussing this because I don't like spreading default values in several places.

Reply:
I'll take another look tomorrow, but I am pretty sure it's necessary

Reply:
Tried it again and it's necessary for the non-Quarkus tests